radiography/src/main/java/radiography/ViewFilter.kt

```diff
@@ -1,18 +1,16 @@
 package radiography
 
-import android.view.View
-
 /**
  * Used to filter out views from the output of [Radiography.scan].
  */
 interface ViewFilter {
   /**
    * @return true to keep the view in the output of [Radiography.scan], false to filter it out.
    */
-  fun matches(view: View): Boolean
+  fun matches(node: Any): Boolean
```

Comment:
sort of a cognitive mitmatch that we have a "ViewFilter" but the param is named node. Either keep the name "view" or rename the filter interface? Not sure "node" is very clear though.

Reply:
Oops, leftover from more invasive rename. I'll revert back to `view`.